addons/pylons/XEH_postInit.sqf

```diff
@@ -0,0 +1,53 @@
+#include "script_component.hpp"
+
+["ace_settingsInitialized", {
+    if (!GVAR(enabled)) exitWith {};
+
+    private _filter = "isClass (_x >> 'Components' >> 'TransportPylonsComponent') && {(getNumber (_x >> 'scope')) > 0}";
+    GVAR(aircraftWithPylons) = (_filter configClasses (configFile >> "CfgVehicles")) apply {configName _x};
+    {
+        [_x, "init", {
+            params ["_aircraft"];
+
+            private _loadoutAction = [
+                QGVAR(loadoutAction),
+                localize LSTRING(ConfigurePylons),
+                "",
+                {[_target] call FUNC(showDialog)},
+                {
+                    private _vehicles = _target nearObjects ["LandVehicle", GVAR(searchDistance) + 10];
+                    private _filter = ["transportAmmo", QEGVAR(rearm,defaultSupply)] select (["ace_rearm"] call EFUNC(common,isModLoaded));
+                    private _rearmVehicles = {([configFile >> "CfgVehicles" >> typeOf _x >> _filter, "number", 0] call CBA_fnc_getConfigEntry) > 0} count _vehicles;
```

Comment:
`_rearmVehicles` -> `_rearmVehiclesCount`.
Also no need to use `CBA_fnc_getConfigEntry` because `getNumber` returns 0 by default.